src/pages/ProductList/Filter/FilterList.js

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect, useState } from 'react';
+import Filter from './Filter';
+import styled from 'styled-components';
+
+function FilterList() {
+  const [filterList, setFilterList] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/filter.json')
+      .then(res => res.json())
+      .then(res => {
+        setFilterList(res);
+      });
+  }, []);
+
+  return (
+    <FilterListWrap>
+      {filterList &&
+        filterList.map(filter => {
+          return (
+            <li key={filter.id}>
+              <Filter
+                key={filter.id}
+                name={filter.name}
+                option={filter.option}
+              />
+            </li>
+          );
+        })}
+    </FilterListWrap>
```

Comment:
이렇게도 할 수 있다~ 정도로 참고용 입니다~
```suggestion
    <FilterListWrap>
      {filterList &&
        filterList.map(({id,name,option}) => {
          return (
            <li key={id}>
              <Filter
                key={id}
                name={name}
                option={option}
              />
            </li>
          );
        })}
    </FilterListWrap>
```

매개변수자리에서 바로 구조분해할당 할 수 있습니다